src/modules/scraper/adapter/messaging/TrackFillEventConsumer.ts

```diff
@@ -0,0 +1,167 @@
+      capitalFeePct,
+      capitalFeeTotal: new BigNumber(capitalFeePct).dividedBy(100).multipliedBy(fromAmounts.formattedAmount).toFixed(),
+      capitalFeeTotalUsd: capitalFeeUsd,
+      fillAmount: fillAmounts.formattedAmount,
+      fillAmountUsd: fillAmounts.formattedAmountUsd,
+      fromAmount: fromAmounts.formattedAmount,
+      fromAmountUsd: fromAmounts.formattedAmountUsd,
+      fromChainId: String(deposit.sourceChainId),
+      fromChainName: sourceChainInfo.name,
+      fromTokenAddress: deposit.tokenAddr,
+      isAmountTooLow: false,
+      lpFeePct: formattedLpFeeValues.pct,
+      lpFeeTotal: formattedLpFeeValues.total,
+      lpFeeTotalUsd: formattedLpFeeValues.totalUsd,
+      NetworkFeeNative: fee,
+      NetworkFeeNativeToken: destinationChainInfo.nativeSymbol,
+      NetworkFeeUsd: feeUsd,
+      recipient: deposit.recipientAddr,
+      referralProgramAddress: deposit.referralAddress || "-",
+      relayFeePct: formattedRelayFeeValues.pct,
+      relayFeeTotal: formattedRelayFeeValues.total,
+      relayFeeTotalUsd: formattedRelayFeeValues.totalUsd,
+      relayGasFeePct: gasFeePct,
+      relayGasFeeTotal: new BigNumber(gasFeePct).dividedBy(100).multipliedBy(fromAmounts.formattedAmount).toFixed(),
+      relayGasFeeTotalUsd: feeUsd,
+      routeChainIdFromTo: `${deposit.sourceChainId}-${deposit.destinationChainId}`,
+      routeChainNameFromTo: `${sourceChainInfo.name}-${destinationChainInfo.name}`,
+      sender: deposit.depositorAddr,
+      succeeded: true,
+      timeFromTransferSignedToTransferCompleteInMilliseconds: String(
+        DateTime.fromISO(fillTx.date).diff(DateTime.fromJSDate(deposit.depositDate)).as("milliseconds"),
+      ),
+      toAmount: new BigNumber(fromAmounts.formattedAmount).minus(formattedBridgeFeeValues.total).toFixed(),
+      toAmountUsd: new BigNumber(fromAmounts.formattedAmountUsd)
+        .minus(formattedBridgeFeeValues.totalUsd)
+        .multipliedBy(depositTokenPriceUsd)
+        .toFixed(),
+      toChainId: String(deposit.destinationChainId),
+      toChainName: destinationChainInfo.name,
+      tokenSymbol: deposit.token.symbol,
+      totalBridgeFee: formattedBridgeFeeValues.total,
+      totalBridgeFeePct: formattedBridgeFeeValues.pct,
+      totalBridgeFeeUsd: formattedBridgeFeeValues.totalUsd,
+      totalFilledAmount: totalFilledAmounts.formattedAmount,
+      totalFilledAmountUsd: totalFilledAmounts.formattedAmountUsd,
+      toTokenAddress: utils.getAddress(destinationToken),
+      transactionHash: fillTx.hash,
+      transferCompleteTimestamp: String(DateTime.fromISO(fillTx.date).toMillis()),
+      transferQuoteBlockNumber: String(fillTxBlock.blockNumber),
+    });
+  }
+
+  private async getFillTxNetworkFee(destinationChainId: number, fillTxHash: string) {
+    const destinationChainProvider = this.providers.getProvider(destinationChainId);
+    const destinationChainInfo = chainIdToInfo[destinationChainId];
```

Comment:
destinationChainInfo might be `undefined`

Reply:
https://github.com/across-protocol/scraper-api/pull/186/commits/a310e99351d2e2e628674b4706d311c914965312